src/test/java/Homework16.java

```diff
@@ -0,0 +1,24 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+public class Homework16 extends BaseTest {
+
+    @Test
+    public void Navigation(){
```

Comment:
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.

For example  `public void navigation()`

Reply:
hi Alex,
thank you! hw updated.
![image](https://user-images.githubusercontent.com/115744919/210440523-01213eba-3d70-42ce-bc6f-dc34c806d7ec.png)